### PR TITLE
Create namespace while installing helm chart by default

### PR DESCRIPTION
**Pull request by k4kratik**
addresses #87 

also I have made RollingUpdate as default Deployment Strategy to avoid any downtime while deploying Pyroscope.

### CODE REVIEW

**Comment by k4kratik** [comment]
I wanted to ask, Why we didn't set the `deploymentStrategy` as `RollingUpdate` ? 
Was it the multi-attach error of PVC?

File: chart/pyroscope/values.yaml [hunk 1]
```diff
@@ -120,7 +120,7 @@ ingress:
 
 # -- Deployment strategy
 deploymentStrategy:
-  type: Recreate
+  type: RollingUpdate
```
Comment (kolesnikovae) [inline chart/pyroscope/values.yaml:123]: Right, use of RWO PVC makes `RollingUpdate` infeasible. Should we maybe make the strategy conditional? Like, it defaults to `RollingUpdate` but if persistence is enabled, force use of `Recreate`

**Review by kolesnikovae — COMMENTED** [review]
Thank you for the contribution @k4kratik!

Also, could you please [run helm-docs and bump the chart version](https://github.com/pyroscope-io/helm-chart/blob/main/CONTRIBUTING.md#technical-requirements)?

File: chart/pyroscope/values.yaml (hunk 1 above)
Comment (k4kratik) [reply chart/pyroscope/values.yaml]: Hi @kolesnikovae 
Thanks for the response. I think I will address this in another PR. 
I have also asked in the [Slack Channel](https://pyroscope.slack.com/archives/C01FJRYENPQ/p1669099704012779), let's discuss it there.

**Comment by kolesnikovae** [comment]
I'm afraid some of the formatting changes broke `helm-doc` – @k4kratik could you check the CI job log please?

File: chart/pyroscope/values.yaml [hunk 2]
```diff
@@ -68,7 +69,8 @@ podSecurityContext:
   fsGroup: 101
 
 # -- Deployment securityContext
-securityContext: {}
+securityContext:
+  {}
```
Comment (kolesnikovae) [inline chart/pyroscope/values.yaml:73]: If there is no strong reason to change the format, let's adhere widely spread convention:
```suggestion
securityContext: {}
```

Comment (k4kratik) [reply chart/pyroscope/values.yaml]: Hi @kolesnikovae 
Thanks, I have reverted back those changes.

**Review by kolesnikovae — APPROVED** [review]

**Comment by kolesnikovae** [comment]
Thank you @k4kratik, nice job!

**Comment by k4kratik** [comment]
Thanks! @kolesnikovae

**Pull request merged**